spring/spring4/pom.xml

```diff
@@ -126,12 +126,27 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
             <scope>runtime</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
             <scope>runtime</scope>
-            <optional>true</optional>
```

Comment:
@danberindei - Did you mean to remove the optional tag for log4j2? Since it's no longer optional, projects that declare a dependency on infinispan-spring4 end up transitively pulling in log4j2 as a runtime dependency, even though they may not want to use log4j2 as their logging framework.